src/app/shared/phone-validator/phone.validator.ts

```diff
@@ -0,0 +1,16 @@
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+import { PhoneNumberUtil, PhoneNumber } from 'google-libphonenumber';
+
+const phoneNumberUtil = PhoneNumberUtil.getInstance();
+
+export function PhoneNumberValidator(regionCode: string = undefined): ValidatorFn {
```

Comment:
We don't need undefined for initialization.

Reply:
fixed